### PR TITLE
Remove the second 'version' declaration

### PR DESCRIPTION
**Pull request by belov38**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/cluster-providers/mng-cluster-provider.md [hunk 1]
```diff
@@ -8,8 +8,7 @@ The `MngClusterProvider` allows you to provision an EKS cluster which leverages
 const props: MngClusterProviderProps = {
     minSize: 1,
     maxSize: 10,
-    desiredSize: 4,
-    version: "auto"
```
Comment (elamaran11) [inline docs/cluster-providers/mng-cluster-provider.md:12]: Why would this have to be removed?

**Review by elamaran11 — COMMENTED** [review]
@belov38 Thankyou for the PR, but unsure why we should remove this from the `MngClusterProviderProps`.

**Review by elamaran11 — APPROVED** [review]
LGTM

**Review by shapirov103 — APPROVED** [review]
LGTM

**Pull request merged**